### ISSUE TITLE
France: Historic Data

### ISSUE THREAD

**Issue by tmtmtmtm** [everypolitician/everypolitician-data#448]
French Wikipedia has information galore: 
  https://fr.wikipedia.org/wiki/Cat%C3%A9gorie:D%C3%A9put%C3%A9_de_la_XIVe_l%C3%A9gislature_de_la_Ve_R%C3%A9publique

**Comment by briatte** [comment]
France a wonderful [official historic database](http://www.assemblee-nationale.fr/sycomore/) for the lower chamber, which I've scraped [here](https://github.com/briatte/parlement/blob/master/sponsors-an.r).

The upper chamber does not publish senator profiles as part of their [open data files](http://data.senat.fr/), but [this page](http://www.senat.fr/anciens-senateurs-5eme-republique/index.html) (Fifth Republic) and [this page](http://www.senat.fr/senateurs/senatl.html) (current senators, IIRC) can be scraped [pretty easily](https://github.com/briatte/parlement/blob/master/sponsors-se.r).

More historical pages:

http://www.senat.fr/senateurs-4eme-republique/index.html
http://www.senat.fr/senateurs-3eme-republique/index.html
http://www.senat.fr/senateurs-2nd-empire/index.html
http://www.senat.fr/senateurs-communaute/index.html

[Regards Citoyens](https://github.com/regardscitoyens/) has more complete data for the recent legislatures of both chambers.

**Comment by tmtmtmtm** [comment]
@briatte I saw that you also had a fork of https://github.com/SocieteCitoyenne/FranceData — did you find that useful?

**Comment by briatte** [comment]
@tmtmtmtm No, I have not found it useful. It's basically Scrapy code to get the same objects that you can get as XML or JSON dumps from http://data.assemblee-nationale.fr/
